ext/opcache/tests/blacklist.phpt

```diff
@@ -8,7 +8,11 @@ opcache.file_update_protection=0
 opcache.file_cache_only=0
 --SKIPIF--
 <?php require_once('skipif.inc'); ?>
-<?php if (substr(PHP_OS, 0, 3) == 'WIN') { die('skip not for Windows'); } ?>
+<?php
+if (substr(PHP_OS, 0, 3) == 'WIN' || PHP_OS_FAMILY === 'Darwin') {
+    die('skip only for Linux');
+}
+?>
```

Comment:
I think it would be better to leave this as one test and write the expectation as `%S/tmp/path`, to account for both possibilities. Possibly with a comment that on macos there will be an additional prefix.

Reply:
OK, I fixed.